dev-docs/adding-data/adding_test_data.md

```diff
@@ -82,13 +88,14 @@ from hab studio:
 
 ensure you have `jq` installed
 
-option one: `infra_service_load_sample_data`
-By default, 50 records of the infra servers and orgs will be added with the prefix `chef-server` and `chef-org` respectively.
-
-option two: `infra_service_load_sample_data -N 100 -S infra-server -O infra-org`
-100 records of the infra servers and orgs will be added, modify the numbers on each of those options to meet your needs.
-
-option three: look through the pinned items in slack channel #automate-infra-views and follow the instructions there (works against running chef server)
+there are three options...
+1. `infra_service_load_sample_data`
```

Comment:
I would also add a warning on (1) and (2) explicitly: "these will result in 'invalid org ID' messages if you drill down to organizations; see (3) to avoid that" -- if I am interpreting what you said for (3) correctly.